README.md

```diff
@@ -4,45 +4,120 @@ The goal is to improve payment confirmation experience with the help of WebAuthn
 
 ![Screenshot](https://raw.githubusercontent.com/rsolomakhin/secure-payment-confirmation/master/payment.png)
 
+The rest of the document is organized into these sections:
+
+- [Creating a credential](#creating-a-credential)
+- [Querying a credential](#querying-a-credential)
+- [Authenticating a payment](#authentication-a-payment)
+
 ## Creating a credential
 
-A new credential type is introduced for `navigator.credentials.create()` that stores the name and the icons of a payment instrument.
+A new `SecurePaymentCredential` credential type is introduced for `navigator.credentials.create()` to bind descriptions of a payment instrument, i.e. a name and an icon, with a vanilla [PublicKeyCredential].
+
+Proposed new spec that extends [Web Authentication]:
+```webidl
+[SecureContext, Exposed=Window]
+interface SecurePaymentCredential : PublicKeyCredential {
```

Comment:
Could we drop the "Secure" bit? Seems redundant.

```suggestion
interface SecurePaymentCredential : PublicKeyCredential {
```

Reply:
Sure!